modules/rostests/winetests/user32/msg.c

```diff
@@ -15569,6 +15577,23 @@ static void test_defwinproc(void)
     ok_sequence(WmRestoreMinimizedOverlappedSeq, "DefWindowProcA(SC_RESTORE):overlapped", TRUE);
     flush_sequence();
 
+    child[0] = CreateWindowExA(0, "TestWindowClass", "1st child",
+                               WS_VISIBLE | WS_CHILD, 0,0,500,100, hwnd, 0, 0, NULL);
+    child[1] = CreateWindowExA(0, "TestWindowClass", "2nd child",
+                               WS_VISIBLE | WS_CHILD, 0,0,500,100, child[0], 0, 0, NULL);
+    child[2] = CreateWindowExA(0, "TestWindowClass", "3rd child",
+                               WS_VISIBLE | WS_CHILD, 0,0,500,100, child[1], 0, 0, NULL);
+    flush_events();
+    flush_sequence();
+    test_context_menu = TRUE;
+    DefWindowProcA(child[2], WM_CONTEXTMENU, 0xcafe, 0);
+    test_context_menu = FALSE;
+    WmContextMenuSeq[0].wParam = (WPARAM)child[2];
+    WmContextMenuSeq[1].wParam = (WPARAM)child[1];
+    WmContextMenuSeq[2].wParam = (WPARAM)child[0];
+    ok_sequence(WmContextMenuSeq, "DefWindowProcA(WM_CONTEXTMENU)", FALSE);
+    DestroyWindow(child[0]);
+
```

Comment:
It would be better to have separate commit for API test changes.

Reply:
> separate commit for API test changes

sure!